src/quo/components/wallet/account_overview/view.cljs

```diff
@@ -113,19 +113,17 @@
           :size   :heading-1
           :style  style/current-value}
          current-value]
-        [rn/view
-         {:style style/row-centered}
-         [:<>
-          (when (seq time-frame-string)
-            [text/text
-             {:weight :medium
-              :size   :paragraph-2
-              :style  (style/bottom-time-text (and (not= :custom time-frame)
-                                                   (seq time-frame-to-string)))}
-             time-frame-string])
-          (when (and (= :custom time-frame)
-                     (seq time-frame-to-string))
-            [custom-time-frame time-frame-to-string])]
+        [rn/view {:style style/row-centered}
+         (when (seq time-frame-string)
```

Comment:
I think it's better to use `string/blank?` to check for string emptiness.

Reply:
I just refactored this code to remove the unnecessary `:<>` there.

But I agree, I'll update it, to improve the whole piece of code :+1:  thanks for spotting it!

Reply:
Hey @ibrkhalil 

I checked the implementation, and I'd prefer keep it as is since it's a bit complex, it's not just about checking a string.

the component receives:
`time-frame`, `time-frame-string`  and `time-frame-to-string`.

then it shadows
`time-frame-string` with `(time-string time-frame time-frame-string)`

and sometimes `time-frame-string` will take the value of the received `time-frame-string`, so I guess the decision to use `seq` was because it's more general.

Since it's a bit hard to track all the flow to make sure this won't create a bug in some cases, I'd prefer to keep it as is, additionally, the `seq` is being used in a translation string, so it won't be `""` or `"      "`.